app/src/ui/changes/changes-list.tsx

```diff
@@ -36,10 +36,10 @@ import { ICommitContext } from '../../models/commit'
 import { RebaseConflictState } from '../../lib/app-state'
 import { ContinueRebase } from './continue-rebase'
 import { enablePullWithRebase } from '../../lib/feature-flag'
-import { ListRow } from '../lib/list/list-row'
 import { Octicon, OcticonSymbol } from '../octicons'
 import { FocusContainer } from '../lib/focus-container'
 import { IStashEntry } from '../../models/stash-entry'
+import classNames = require('classnames')
```

Comment:
```suggestion
import * as classNames from 'classnames'
```

Reply:
I have no clue why Vs Code did that style of import...

Reply:
My gut feeling it's something related to the TS services in VSCode, and `require` is a safer bet for Node projects...